### PR TITLE
release 8.1.0

### PR DESCRIPTION
**Pull request by vchepkov**
#### Pull Request (PR) description
release 8.1.0

### CODE REVIEW

**Comment by dhoppe** [comment]
According to this pull request https://github.com/voxpupuli/puppet-r10k/pull/502, it should be a major release. Please change the version number to 9.0.0.

File: metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "author": "Vox Pupuli",
   "license": "Apache-2.0",
   "name": "puppet-r10k",
-  "version": "8.0.1-rc0",
+  "version": "8.1.0",
```
Comment (dhoppe) [inline metadata.json:5]: According to this pull request #502, it should be a major release. Please change the version number to 9.0.0.

Comment (vchepkov) [reply metadata.json:5]: That was done in previous version, when it was bumped to v8, right?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,12 +4,37 @@ All notable changes to this project will be documented in this file.
...
+- fix webhook generate type functionailty for non-root user [\#512](https://github.com/voxpupuli/puppet-r10k/pull/512) ([vchepkov](https://github.com/vchepkov))
+- allow puppetlabs/inifile 4.x [\#505](https://github.com/voxpupuli/puppet-r10k/pull/505) ([bastelfreak](https://github.com/bastelfreak))
+
+## [v8.0.0](https://github.com/voxpupuli/puppet-r10k/tree/v8.0.0) (2019-10-28)
 
 [Full Changelog](https://github.com/voxpupuli/puppet-r10k/compare/v7.0.0...v8.0.0)
 
 **Breaking changes:**
 
+- Don't manage installation of `git` [\#502](https://github.com/voxpupuli/puppet-r10k/pull/502) ([Andor](https://github.com/Andor))
```
Comment (alexjfisher) [inline CHANGELOG.md:37]: Something has gone a bit wrong.  Please don't merge until I can investigate what.

**Comment by alexjfisher** [comment]
I think we need a modulesync first then a rebase.  This should fix the issue with the changelog generation.

**Comment by vchepkov** [comment]
close in favor of #521 #521

**Pull request closed without merging**